node-repository/src/main/java/com/yahoo/vespa/hosted/provision/NodeList.java

```diff
@@ -277,6 +280,12 @@ private void ensureSingleCluster() {
     /** Returns the nodes of this as a stream */
     public Stream<Node> stream() { return asList().stream(); }
 
+    public static NodeList of(Node ... nodes) {
+        List<Node> nodeList = new ArrayList<>();
+        Collections.addAll(nodeList, nodes);
+        return copyOf(nodeList);
```

Comment:
Can be simplified to `return copyOf(List.of(nodes))`.

Reply:
Thanks, done